projects/angular/src/layout/nav/_responsive-nav.clarity.scss

```diff
@@ -274,43 +252,21 @@
         // TODO: more specificity in classname is preferable to nesting...
         .nav {
           padding-left: 0;
-          @include mixins.css-var(gap, clr-nav-vertical-gap-size, 0, variables.$clr-use-custom-properties);
+          gap: tokens.$cds-global-space-5;
 
           .nav-item {
-            @include mixins.css-var(
-              height,
-              clr-subnav-sm-item-height,
-              variables.$clr_baselineRem_1_5,
-              variables.$clr-use-custom-properties
-            );
+            height: tokens.$cds-global-space-11;
             margin-right: 0;
           }
 
           .nav-link {
-            @include mixins.css-var(
-              padding,
-              clr-subnav-sm-link-padding,
-              0 variables.$clr_baselineRem_0_5 0 variables.$clr_baselineRem_1,
-              variables.$clr-use-custom-properties
-            );
+            padding: vertical-nav-variables.$clr-vertical-nav-item-padding;
 
             width: 100%;
             max-width: 100%;
             overflow: hidden;
             text-overflow: ellipsis;
-            border-radius: 0;
-            @include mixins.css-var(
-              border-top-left-radius,
-              clr-subnav-sm-link-border-radius,
-              variables.$clr-global-borderradius,
-              variables.$clr-use-custom-properties
-            );
-            @include mixins.css-var(
-              border-bottom-left-radius,
-              clr-subnav-sm-link-border-radius,
-              variables.$clr-global-borderradius,
-              variables.$clr-use-custom-properties
-            );
+            border-radius: tokens.$cds-alias-object-border-radius-100 0 0 tokens.$cds-alias-object-border-radius-100;
```

Comment:
Yes, first we used to set border-radius to 0 for each border radius side, then we added custom values for top left and bottom left. 

Now we have it as one liner, it should supposedly have some minimal performance benefits if it is a one liner, and since the shim removal started I think we use one liners wherever possible.